### ISSUE TITLE
internal server error when creating page with title /code/python/decorating class methods

### ISSUE THREAD

**Issue by battlemidget** [claudioc/jingo#80]
jingo 1.2.11

Wanted to create a page under the /code/python namespace and received:

```
Error: git --git-dir=.git --work-tree=. add -Code-Python-Decorate-class-variables.md
error: unknown switch `C'
usage: git add [options] [--] <pathspec>...

    -n, --dry-run         dry run
    -v, --verbose         be verbose

    -i, --interactive     interactive picking
    -p, --patch           select hunks interactively
    -e, --edit            edit current diff and apply
    -f, --force           allow adding otherwise ignored files
    -u, --update          update tracked files
    -N, --intent-to-add   record only the fact that the path will be added later
    -A, --all             add changes from all tracked and untracked files
    --ignore-removal      ignore paths removed in the working tree (same as --no-all)
    --refresh             don't add, only refresh the index
    --ignore-errors       just skip files which cannot be added because of errors
    --ignore-missing      check if - even missing - files are ignored in dry run
```

This could very well be a user error but i'm coming from gollum and that is how i used to create those pages.

Thanks!

**Comment by battlemidget** [comment]
saw the limitation :\, would like to keep this open as a feature request though

**Comment by claudioc** [comment]
Hi, could you please tell me the steps to get to the error? I am not sure about the feature request, but it's not cool to have a crash anyway :) Thanks!

**Comment by battlemidget** [comment]
In the title box when you type something like "Code/Python/decorate class variables" is when it crashes

**Comment by claudioc** [comment]
Closed in 1.3.0
